lib/spandex_tesla.ex

```diff
@@ -44,7 +44,7 @@ defmodule SpandexTesla do
       )
 
     tracer().span_error(
-      %Error{message: error},
+      %Error{message: Atom.to_string(error)},
```

Comment:
The `error` will alway be a string? Or there i some scenario where it can be something else?

Reply:
In fact, this error will always be an Atom, and I always need to pass that as a string to spandex. I had an error sending as an Atom

Reply:
Yeah, makes sense. If we get into a scenario that tesla is publish a error with a type different than atom we can always revisit this point and change the implementation.